includes/REST_API/Media_Hotlinking_Controller.php

```diff
@@ -0,0 +1,290 @@
+<?php
+/**
+ * Class Media_Hotlinking_Controller
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\REST_API;
+
+use Google\Web_Stories\Story_Post_Type;
+use Google\Web_Stories\Traits\Post_Type;
+use Google\Web_Stories\Traits\Types;
+use phpDocumentor\Reflection\Types\Integer;
```

Comment:
Added by mistake?

Reply:
Friendly reminder, this is a draft, I don't want a code review. I requested a functional test to check it is fit for requirements. I will tidy up later.

Reply:
I understand that. Just trying to be helpful. I spotted it and might have forgotten later on, so I wrote it down. Better to point this out instead of having it accidentally slip through.